vcd_cli/role.py

```diff
@@ -21,6 +21,9 @@ def role(ctx):
             
         vcd role list_rights
             Get list of rights associated with a given role.
+        
+        vcd role create role1 description 'Disk: View Properties' 'Provider vDC: Edit' --org myOrg
+            Create a role with zero or more rights in the specified Org or current Org in use.
```

Comment:
The use of 'or' is confusing.
 Create a role with zero or more rights in the specified Org (defaults to current Org in use).

Reply:
fixed.